Builds/linux/README.md

```diff
@@ -88,14 +88,14 @@ git checkout develop
 If you didn't persistently set the `BOOST_ROOT` environment variable to the
 directory in which you compiled boost, then you should set it temporarily.
 
-For example, you built Boost in your home directory `~/boost_1_70_0`, you
+For example, you built Boost in your home directory `~/boost_1_71_0`, you
 would do for any shell in which you want to build:
 
 ```
-export BOOST_ROOT=~/boost_1_70_0
+export BOOST_ROOT=~/boost_1_71_0
 ```
 
-Alternatively, you can add `DBOOST_ROOT=~/boost_1_70_0` to the command line when
+Alternatively, you can add `DBOOST_ROOT=~/boost_1_71_0` to the command line when
```

Comment:
While this file is being touched, this parameter is missing the `-` i.e. `-DBOOST_ROOT=~/boost_1_71_0`
I like copy/pasteable things.

Reply:
done, thanks for the suggestion @legleux !